src/components/avatars/Avatar.stories.js

```diff
@@ -0,0 +1,90 @@
+// import MyButton from './Button.vue';
+import DdAvatar from "./index.vue"
+import { action } from "@storybook/addon-actions"
+
+// More on default export: https://storybook.js.org/docs/vue/writing-stories/introduction#default-export
+export default {
+  title: 'Example/DdAvatar',
+  component: DdAvatar,
+  // More on argTypes: https://storybook.js.org/docs/vue/api/argtypes
+  argTypes: {
+    color: { 
+      description: "Change color value as per the requirement",
+      table:{
+        defaultValue:{
+          summary: "-"
+        }
+      }
+     },
+     altName:{
+      description: "Change title value as per the requirement",
+      table:{
+        defaultValue:{
+          summary: "-"
+        }
+      }
+     },
+     srcLink:{
+      description: "Change title value as per the requirement",
+      table:{
+        defaultValue:{
+          summary: "-"
+        }
+      }
+     },
+    title:{
+      description: "Change title value as per the requirement",
+      table:{
+        defaultValue:{
+          summary: "-"
+        }
+      }
+     },
+    size: {
+      control: { type: 'select' },
+      options: ['mini','small', 'medium', 'large','xLarge',],
+      description: "Accepted value mini / small / medium / large / xLarge ",
+      table:{
+        defaultValue:{
+          summary: "medium"
+        }
+      }
+    },
+    type: {
+      control: { type: 'select' },
+      options: ['default','group',],
+       description: "By default avatar type is color less so change the props default / group ",
+      table:{
+        defaultValue:{
+          summary: "-"
+        }
+      }
+    },
+  },
+};
+
+// More on component templates: https://storybook.js.org/docs/vue/writing-stories/introduction#using-args
+const Template = (args) => ({
+  // Components used in your story `template` are defined in the `components` object
+  components: { DdAvatar },
+  // The story's `args` need to be mapped into the template through the `setup()` method
+  setup() {
+    const onClickMethod =  (() => action("clicked"))
+    return { args, onClickMethod };
+
+  },
+  // And then the `args` are bound to your component with `v-bind="args"`
+  template: '<img-avatar @click="onClickMethod" v-bind="args" />  ',
```

Comment:
template name should be same as the components name you have imported DdAvatar but in template its sill <img-avatar also there is no need of @click in this component remove it

Reply:
Template name changed and also @click component is removed.
<img width="984" alt="image" src="https://user-images.githubusercontent.com/85391382/217446052-bc7204d1-5519-4eda-8d59-0c4e0b5f8e6b.png">